jira/client.py

```diff
@@ -4381,6 +4381,26 @@ def delete_permissionscheme(self, id: str):
         self.permissionschemes.cache_clear()
         return data
 
+    def get_form(self, issue: str, formid: str) -> list[dict]:
+        """For the specified issue, returns the form data as key/value list. (Admin required).
+
+        Args:
+            issue (str): The issue id.
+            formid (str): the ID of the form, default is 1
```

Comment:
how do we enforce this default? It doesn't seem to be set in the function signature.

Reply:
why enforce? maybe it should instead be renamed to "starts with 1". i am no fan of implicit magic that happens somehow. if someone wants the form data, he should know the ID. if he has multiple forms, then it might be 2, 3 or something else.

Reply:
That would be better yes, if we are saying that form IDs at 1 indexed that is different from saying the "default" is 1, as that has specific connotations in function documentations.